microprofile/metrics/src/test/resources/logging.properties

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2021 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Example Logging Configuration File
+# For more information see $JAVA_HOME/jre/lib/logging.properties
+
+# Send messages to the console
+handlers=io.helidon.common.HelidonConsoleHandler,java.util.logging.FileHandler
+
+# HelidonConsoleHandler uses a SimpleFormatter subclass that replaces "!thread!" with the current thread
+java.util.logging.SimpleFormatter.format=%1$tY.%1$tm.%1$td %1$tH:%1$tM:%1$tS %4$s %3$s !thread!: %5$s%6$s%n
+
+# Global logging level. Can be overridden by specific loggers
+.level=INFO
+
+# Component specific log levels
+#io.helidon.webserver.level=INFO
+#io.helidon.config.level=INFO
+#io.helidon.security.level=INFO
+#io.helidon.common.level=INFO
+#io.netty.level=INFO
+#io.helidon.microprofile.metrics.MetricsCdiExtension.level=FINEST
+#io.helidon.microprofile.metrics.Interceptor*.level=FINEST
+
+#java.util.logging.FileHandler.level=FINEST
+#java.util.logging.FileHandler.formatter=java.util.logging.SimpleFormatter
+#io.helidon.common.HelidonConsoleHandler.level=FINEST
+
+org.jboss.weld.Bootstrap.level = FINEST
```

Comment:
Please use max. INFO for tests. Probably left uncommented when troubleshooting...

Reply:
Fixed.